content/release-10-16-0/platform-10-16-0-bundle/10-16-0-159.md

```diff
@@ -0,0 +1,53 @@
+---
+weight: 38
+title: Release 10.16.0.159
+layout: bundle
+---
+
+<!--10.16.0.122-10.16.0.159-->
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Authentication</td>
+<td>The performance of the first user requests sent after node restart has been improved. The issue was causing problems in rare cases where the number of all global roles in a tenant was close to but not exceeding 100, and many thousands of devices were concurrently trying to authorize MQTT connections on the restarted node.</td>

```

Comment:
this talks about an improvement and then proceeds to talk about an "issue", which is a strange continuation. Maybe rephrase to "Before, there were problems in rare cases..."